BMM.Core/Constants/LoggingEvents.cs

```diff
@@ -0,0 +1,7 @@
+namespace BMM.Core.Constants
+{
+    public class LoggingEvents
+    {
+        public const string ViewModelOpenedFormat = "{0} opened";
```

Comment:
There's already a similar class: BMM.Core.Implementations.Analytics.Event
I think I like your namespace better. Nevertheless they should be merged.